stubs/CoreGenericFunctions.phpstub

```diff
@@ -334,6 +334,22 @@ function fclose(&$stream) : bool
 {
 }
 
+/**
+ * @psalm-pure
+ * @template T as string
+ * @param T $string
+ * @return T
+ */
+function sodium_bin2base64(string $string, int $id): string
+
+/**
+ * @psalm-pure
+ * @template T as string
+ * @param T $string
+ * @return T
+ */
+function sodium_bin2hex(string $string): string {}
```

Comment:
This isn't quite right, passing a `numeric-string` shouldn't return a `numeric-string`. Please use a conditional return type instead.